test/utils/padder/padder.go

```diff
@@ -204,10 +206,14 @@ func (p *Padder) Clean() error {
 	if errors.IsNotFound(err) {
 		err = nil
 	}
-	p.targetedNodes = []string{}
+	p.PaddedNodes = []string{}
 	return err
 }
 
+func (p *Padder) GetPaddedNodes() []string {
```

Comment:
why, considering the field is already public?

Reply:
The field shouldn't be public. my bad